client/blocks/login/two-factor-authentication/waiting-notification-approval.jsx

```diff
@@ -62,7 +62,7 @@ class WaitingTwoFactorNotificationApproval extends Component {
 	componentWillReceiveProps( nextProps ) {
 		if ( ! this.props.pushSuccess && nextProps.pushSuccess ) {
 			const { translate } = this.props;
-			this.props.successNotice( translate( 'Verified successfully!' ) );
+			this.props.successNotice( translate( 'Logging in…' ) );
```

Comment:
The capitalization feels a little awkward to me, since if we wrote this from scratch we wouldn't capitalize `In`... I guess it's worth not having to retranslate though.